run_webcore.sh

```diff
@@ -1,9 +1,245 @@
 #!/usr/bin/env bash
 
-# This is a temporary script to run WebCORE, since no command line option is available for now
+JAVA_BIN="/usr/local/jdk-11.0.1/bin/java"  # must be Java 11
```

Comment:
This script can be used to run WebCORE from the terminal, after having built it in Eclipse. In the future, this script should be migrated to one of the backend repos